d3a_api_client/redis_client_base.py

```diff
@@ -71,8 +72,8 @@ def register(self, is_blocking=False):
         logging.info(f"Trying to register to {self.area_id} as client {self.client_id}")
         if self.is_active:
             raise RedisAPIException(f'API is already registered to the market.')
-
-        data = {"name": self.client_id, "transaction_id": str(uuid.uuid4())}
+        self.device_uuid = str(uuid.uuid4())
+        data = {"name": self.client_id, "transaction_id": self.device_uuid}
```

Comment:
the device uuid is not the same as transaction_id.

Reply:
`self.device_uuid` is needed for aggregator-device mapping, so why not use `transaction_id ` at the time of instantiating since it already unique.

Reply:
I dont agree. It is not a big effort to create anther uuid for the transaction_id. I predict that this will lead to confusion in the future. So please revert. Thanks!